### DISCUSSION TITLE
is it possible to use Apache AGE with newer versions of Postgres >11 ?

### DISCUSSION THREAD

**Discussion by leonh** [apache/age#81] [General]
This project looks very interesting particularly the possibility of hybrid Cypher/SQL queries. 

....also very interested in the changes to JSON queries coming in Postgres 14.  https://blog.crunchydata.com/blog/better-json-in-postgres-with-postgresql-14 

Is there a possibility to use Apache AGE with newer Postgres versions?

**Comment by jrgemignani** [comment]
More recent versions of PostgreSQL will be supported in the near future.  However, right now our focus is on finishing AGE for PostgreSQL 11. My understanding is that PostgreSQL 12 has significant changes that will break the current implementation of AGE.

Hope that is helpful.
